commands/rm.go

```diff
@@ -47,7 +47,7 @@ func cmdRm(c CommandLine, api libmachine.API) error {
 		}
 	}
 
-	if len(errorOccurred) > 0 && !force {
```

Comment:
This is the fix for https://github.com/rancher/rancher/issues/42420

Reply:
Based on the offline discussion, the fix for  https://github.com/rancher/rancher/issues/42420 will be handled in a separate PR as the proper fix will be complex.